source/_integrations/shelly.markdown

```diff
@@ -113,6 +113,12 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Action driven battery powered devices (CoIoT) (initial support)
+For action driven Shelly battery powered devices such as the Shelly Motion it is required that the "CoIoT" option is set so that the sensor can send action events, such as "motion detection" to Home Assistant.
```

Comment:
The "CoIoT" is required for the sensor to work, not only for actions. It is also need to be set before adding the device to Home Assistant, I think it worth mentioning this.
I am not sure how to correctly call this new category, for now it is only Shelly Motion, but it is planned to add this support for other devices also.
Maybe change the header:
## Silabs based powered devices (CoIoT) (initial support)